Jupyter/profit_multistrat_plot.ipynb

```diff
@@ -0,0 +1,4960 @@
+      "2018-05-13 21:57:14,957 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:14,957 - freqtrade.exchange - INFO - Instance is running with dry_run enabled\n",
+      "2018-05-13 21:57:14,967 - freqtrade.exchange - INFO - Using Exchange \"Binance\"\n",
+      "2018-05-13 21:57:26,355 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:26,994 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:26,994 - freqtrade.exchange - INFO - Instance is running with dry_run enabled\n",
+      "2018-05-13 21:57:27,009 - freqtrade.exchange - INFO - Using Exchange \"Binance\"\n",
+      "2018-05-13 21:57:38,340 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "d:\\work\\finances\\freqtrade\\myfreqtrade\\freqtrade\\user_data\\strategies\\ReinforcedQuickie.py:188: FutureWarning:\n",
+      "\n",
+      "how in .resample() is deprecated\n",
+      "the new syntax is .resample(...)..apply(<func>)\n",
+      "\n",
+      "2018-05-13 21:57:40,765 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:40,765 - freqtrade.exchange - INFO - Instance is running with dry_run enabled\n",
+      "2018-05-13 21:57:40,765 - freqtrade.exchange - INFO - Using Exchange \"Binance\"\n",
+      "2018-05-13 21:57:49,061 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:49,916 - freqtrade.strategy.resolver - INFO - Override strategy 'ticker_interval' with value in config file: 5m.\n",
+      "2018-05-13 21:57:49,932 - freqtrade.exchange - INFO - Instance is running with dry_run enabled\n",
+      "2018-05-13 21:57:49,932 - freqtrade.exchange - INFO - Using Exchange \"Binance\"\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.plotly.v1+json": {
+       "data": [
+        {
+         "line": {
+          "color": "rgb(200, 200, 200)",
+          "width": 4
+         },
+         "name": "",
+         "type": "scatter",
+         "x": [
+          "2018-01-30 06:50:00",
+          "2018-03-16 20:10:00"
+         ],
+         "y": [
+          100,
+          100
+         ]
+        },
+        {
+         "name": "BinHV27",
+         "type": "scatter",
+         "x": [
+          "2018-01-30 06:50:00",
+          "2018-02-01 09:05:00",
+          "2018-02-01 12:20:00",
+          "2018-03-16 20:10:00"
+         ],
+         "y": [
+          100,
+          49.917893,
+          24.872442285683512,
+          49.84308296330628
+         ]
+        },
+        {
+         "name": "BinHV45",
+         "type": "scatter",
+         "x": [
+          "2018-01-05 04:20:00",
```

Comment:
This script seems to hardcode huge amount of data instead of loading it from tickers or our test data sets. It will be a nightmare to keep up to date.